### DISCUSSION TITLE
⭐️ Starstruck Exchange ⭐️

### DISCUSSION THREAD

**Discussion by xommmax** [Schweinepriester/github-profile-achievements#752] [Q&A]
HI everyone!
[Flutter Guidelines](https://github.com/4lfant/flutter-guidelines) repository contains my personal recommendations for policies, procedures, and standards regarding mobile development. I collected all my knowledge in this document.
Please star it 🥰
After you starred leave a comment here with your repo and I will star it in response

**Comment by Schweinepriester** [comment]
See https://github.com/Schweinepriester/github-profile-achievements/discussions/722
